package.json

```diff
@@ -4,12 +4,18 @@
   "description": "WordPress Theme framework",
```

Comment:
These scripts are more or less c+p from BS right?

Reply:
Exactly, we're thinking it makes the most sense to follow what they do in BS4/5. I just trimmed a lot of them down and also tried to bring the naming conventions closer to what we had before ('gulp watch' becomes 'npm run watch')